File: hidden-api/src/main/java/android/content/pm/ApplicationInfoHidden.java
package android.content.pm;

import dev.rikka.tools.refine.RefineAs;

@RefineAs(ApplicationInfo.class)
public class ApplicationInfoHidden {

  public static final int PRIVATE_FLAG_HIDDEN = 1;

  public String primaryCpuAbi;
  public int privateFlags;
}

File: hidden-api/src/main/java/android/content/pm/ApplicationInfoHidden.kt
package android.content.pm

import dev.rikka.tools.refine.RefineAs

@RefineAs(ApplicationInfo::class)
class ApplicationInfoHidden {

  companion object {
    const val PRIVATE_FLAG_HIDDEN = 1
  }

  var primaryCpuAbi: String? = null
  var privateFlags: Int = 0
}